clients/benchmarks/CMakeLists.txt

```diff
@@ -66,60 +66,51 @@ target_include_directories( rocblas-bench
 set( BLIS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/build/deps/blis/include/blis )
 set( BLIS_LIBRARY ${CMAKE_SOURCE_DIR}/build/deps/blis/lib/libblis.so )
 
-
-if( OS_ID_rhel OR OS_ID_centos OR OS_ID_sles)
+if( OS_ID_rhel OR OS_ID_sles OR OS_ID_centos)
     if( OS_ID_rhel OR OS_ID_centos)
         # defer OpenMP include as search order must come after clang
-        set( XXX_OPENMP_INCLUDE_DIR /opt/rh/devtoolset-7/root/usr/lib/gcc/x86_64-redhat-linux/7/include )
```

Comment:
XXX was workaround to reorder includes, see later comment.  Good to remove now if it works with latest compiler and rocm install.